kse/src/org/kse/gui/dialogs/resources_fr.properties

```diff
@@ -114,6 +114,9 @@ DGenerateKeyPairCert.NameValueReq.message           = Veuillez pr\u00E9ciser un
 DGenerateKeyPairCert.SerialNumberNonZero.message    = Le num\u00E9ro de s\u00E9rie doit \u00EAtre sup\u00E9rieur \u00E0 z\u00E9ro.
 DGenerateKeyPairCert.SerialNumberNotInteger.message = Le num\u00E9ro de s\u00E9rie doit \u00EAtre un entier.
 DGenerateKeyPairCert.ValReqSerialNumber.message     = Veuillez pr\u00E9ciser un num\u00E9ro de s\u00E9rie.
+DGenerateKeyPairCert.jbTransferNameExt.mnemonic     = T
+DGenerateKeyPairCert.jbTransferNameExt.text         = Transf\u00E9rer nom et ext.
+DGenerateKeyPairCert.jbTransferNameExt.tooltip      = Transf\u00E9rer nom et extensions \u00E0 partir d'un certificat
```

Comment:
Hello @jgrateron,

Here is a minor change _(and it is due to my fault!)_

Could you change this line:
```properties
DGenerateKeyPairCert.jbTransferNameExt.tooltip      = Transf\u00E9rer nom et extensions \u00E0 partir d'un certificat
```
By:
```properties
DGenerateKeyPairCert.jbTransferNameExt.tooltip      = Transf\u00E9rer nom et extensions \u00E0 partir d\u2019un certificat
```


@kaikramer: We let you do the German version... 😉

Regards.

Reply:
Fixed, regards.